### PR TITLE
[SPARK-32171][SQL][DOCS] Change file locations for use db and refresh table

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?

docs/sql-ref-syntax-qry-select-usedb.md -> docs/sql-ref-syntax-ddl-usedb.md
docs/sql-ref-syntax-aux-refresh-table.md -> docs/sql-ref-syntax-aux-cache-refresh-table.md

### Why are the changes needed?
usedb belongs to DDL. Its location should be consistent with other DDL commands file locations
similar reason for refresh table

### Does this PR introduce _any_ user-facing change?
before change, when clicking USE DATABASE, the side bar menu shows select commands  
Screen Shot 2020-07-04 at 9 05 35 AM

after change, when clicking USE DATABASE, the side bar menu shows DDL commands
Screen Shot 2020-07-04 at 9 06 06 AM

before change, when clicking refresh table, the side bar menu shows Auxiliary statements
Screen Shot 2020-07-04 at 9 30 40 AM

after change, when clicking refresh table, the side bar menu shows Cache statements
Screen Shot 2020-07-04 at 9 35 21 AM

### How was this patch tested?
Manually build and check

### CODE REVIEW

**Comment by huaxingao** [comment]
cc @dilipbiswal

**Comment by dilipbiswal** [comment]
LGTM

**Review by viirya — APPROVED** [review]

File: docs/_data/menu-sql.yaml [hunk 1]
```diff
@@ -139,7 +139,7 @@
             - text: REPAIR TABLE
               url: sql-ref-syntax-ddl-repair-table.html
             - text: USE DATABASE
-              url: sql-ref-syntax-qry-select-usedb.html
+              url: sql-ref-syntax-ddl-usedb.html
```
Comment (maropu) [inline docs/_data/menu-sql.yaml:142]: Ah, I see. Nice catch!

**Review by maropu — APPROVED** [review]

**Comment by huaxingao** [comment]
Thank for reviewing. I will use this PR to try the merge script. 
Seems Jenkins is down. Can I merge without waiting for test result (since this is a docs PR)? Or should I wait?

**Comment by huaxingao** [comment]
@ulysses-you please change the doc links in https://github.com/apache/spark/pull/28840 after this PR is merged. Thanks!

**Comment by maropu** [comment]
This is just a minor doc fix, so merging it looks okay for your practice.

**Comment by maropu** [comment]
Could you please merge this into branch-3.0?

**Comment by huaxingao** [comment]
@maropu merge into both master and branch 3.0, right?

**Comment by huaxingao** [comment]
Merged to master/3.0. Thanks everyone!

**Comment by ulysses-you** [comment]
@huaxingao get it, thanks !

**Pull request closed without merging**